bam_markdup.c

```diff
@@ -33,15 +33,38 @@ DEALINGS IN THE SOFTWARE
 #include <ctype.h>
 #include <time.h>
 #include <sys/stat.h>
+#include <math.h>
 #include "htslib/thread_pool.h"
 #include "htslib/sam.h"
 #include "sam_opts.h"
 #include "samtools.h"
+#include "htslib/cram.h"
```

Comment:
This can be removed as `stringify_argv()` is now in `htslib/sam.h`

Reply:
Okay.